tests/basic/pos/ReflExt01.hs

```diff
@@ -0,0 +1,16 @@
+-- | test reflection away from the source, using the unfoldings of the interface files
+
+{-@ LIQUID "--ple"         @-}
+{-@ LIQUID "--reflection" @-}
+
+module ReflExt01 where
```

Comment:
I think we also would need some tests to observe the error messages that are produced when the unfoldings aren't available.

Reply:
Addressed in a2bd4dd698e671a1b041909d20bc133cd73089c6